tools/check/go.mod

```diff
@@ -21,6 +20,7 @@ require (
 	go.uber.org/atomic v1.9.0 // indirect
 	golang.org/x/mod v0.5.1 // indirect
 	golang.org/x/tools v0.1.6 // indirect
+	gotest.tools/gotestsum v1.7.0 // indirect
```

Comment:
why we use this now?

Reply:
CI tema found  that some log parsing is missing by using `go-junit-report`, so they recommend use `gotestsum` instead of `go-junit-report`